pulp_rpm/app/tasks/publishing.py

```diff
@@ -273,6 +275,7 @@ def create_rempomd_xml(packages, publication, extra_repomdrecords, sub_folder=No
     """
     cwd = os.getcwd()
     repodata_path = REPODATA_PATH
+    contain_modular = False
```

Comment:
I think the verb is usually in the third form for boolean variables.
How about `contains_modules` or `has_modules`?